packages/editor/src/components/site-settings-panel/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import BlogTitlePanel from './blog-title-panel';
+import PostsPerPagePanel from './posts-per-page-panel';
+import DiscussionPanel from './discussion-panel';
+import { store as editorStore } from '../../store';
+
+export default function SiteSettingsPanel() {
```

Comment:
Can we avoid this temporary component and just move the rendering of the three components into the PostSummary component instead (to follow what's done for all the other components)

Reply:
I was hesitant about this because now we render these controls only when editing home or index template, where they could reused in a different part of the UI at the future. If we have these checks in the standalone components it will make reusability harder. Both approaches though are hypothetical right now and we can go either way, it's just that I was considering more possible to reuse these panels in a settings page or something.

I can make this change though if you have strong opinion